src/webview/common-ext/createComponentHelpers.ts

```diff
@@ -109,6 +109,33 @@ export function validateComponentName(name: string): string {
     return validationMessage;
 }
 
+/**
+ * Returns the validation message if the component name is invalid, and undefined otherwise.
+ *
+ * @param name the port number to validate
+ * @returns the validation message if the component name is invalid, and undefined otherwise
+ */
+export function validatePortNumber(portNumber: string): string {
+    let validationMessage: string;
+    if (portNumber.trim().length > 0) {
+        validationMessage = NameValidator.lengthName(
```

Comment:
If I want to set the port to `20`, then I have to enter `00020`. Is it possible to validate the port based only on the value of the number?